src/TypeDefinitions.js

```diff
@@ -46,6 +46,7 @@ export type PagerRendererProps<T> = PagerCommonProps<T> & {
 export type PagerCommonProps<T> = {
   animationEnabled?: boolean,
   swipeEnabled?: boolean,
+  bouncesEnabled?: boolean,
```

Comment:
I think this should be in `PagerScroll`'s props because it's not a common prop. We also need to add it to the `propTypes`

Reply:
I can't only set `bounces` prop for `PagerScroll` because this prop should be assigned for `TabView`.

`TabFlow`'s type is 
```
type Props<T> = PagerCommonProps<T> &
  PagerExtraProps & {
    navigationState: NavigationState<T>,
    onIndexChange: (index: number) => mixed,
    initialLayout?: Layout,
    renderPager: (props: *) => React.Node,
    renderScene: (props: SceneRendererProps<T> & Scene<T>) => React.Node,
    renderTabBar: (props: SceneRendererProps<T>) => React.Node,
    tabBarPosition: 'top' | 'bottom',
    useNativeDriver?: boolean,
    style?: ViewStyleProp,
  };
```
And `PagerScoll`'s type is 
```
type Props<T> = PagerRendererProps<T>

export type PagerRendererProps<T> = PagerCommonProps<T> & {
  layout: Layout & {
    measured: boolean,
  },
  navigationState: NavigationState<T>,
  panX: Animated.Value,
  offsetX: Animated.Value,
  jumpTo: (key: string) => mixed,
  useNativeDriver: boolean,
  children: Node,
};
```

The shared type of `TabView` and `PagerScroll` is `PagerCommonProps`. I can only set `bounces` type in `PagerCommonProps` to pass flowtype validation.

But Android user will not be affect by this prop. And I have added `bounces` prop to `PropTypes` file.